app/views/admin/feedback_messages/_feedback_message.html.erb

```diff
@@ -90,7 +90,7 @@
             <% if feedback_message_emails.any? %>
               <% feedback_message_emails.each do |email| %>
                 <div class="email__container">
-                  <p class="to__subject">Type: <%= email.utm_campaign.capitalize %></p>
+                  <p class="to__subject">Type: <%= email&.utm_campaign&.capitalize %></p>
```

Comment:
Hm, if `utm_campaign` is `nil`, do we even want to render `<%= email&.utm_campaign %>` and its surrounding `<p>` tag? In other words, if there is no `utm_campaign`, what are we rendering? Unless I'm misunderstanding, I _think_ we'd render: `Type: ` with nothing following it. Do we intentionally want to render that, or should we not render this `<p>` tag at all? 🤔